packages/x-components/src/components/animations/collapse-width.vue

```diff
@@ -26,7 +27,9 @@
   @Component({
     mixins: [createCollapseAnimationMixin('width')]
```

Comment:
Shouldn't we add `inheritAttrs` here too?

```suggestion
    mixins: [createCollapseAnimationMixin('width')],
    inheritAttrs: true
```